src/main/java/VladMaltsev/weatherapp/util/dtoconversion/MappingDTOAndClass.java

```diff
@@ -0,0 +1,21 @@
+package VladMaltsev.weatherapp.util.dtoconversion;
+
+import org.modelmapper.ModelMapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MappingDTOAndClass {
```

Comment:
add more explanation and more descriprive names. I don't understand what is mapped to what and why this is required.